### PR TITLE
wording clarifications

### PR DESCRIPTION
**Pull request by liamn-myob**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,13 +23,13 @@ provider. It should implement the following:
 ]
 ```
 
-- tests or a test suite; the type of testing is up to you
+- tests or a test suite.s
```
Comment (felix) [inline README.md]: Typo

File: README.md [hunk 2]
```diff
@@ -23,13 +23,13 @@ provider. It should implement the following:
...
 
 ## Fit and Finish
 
 Once the application has been written, continue with the following additions:
 
 - provide a means of packaging your application as a single deployable artifact which encapsulates its dependencies
-- create a pipeline that builds your application on each commit; Travis or similar, for example
+- create a pipeline that builds your application on each commit; Travis, Github Actions or similar
```
Comment (felix) [inline README.md]:
```suggestion
- create a pipeline that builds your application on each commit; Travis, Github Actions, or similar
```

File: README.md [hunk 3]
```diff
@@ -23,13 +23,13 @@ provider. It should implement the following:
...
 - describe or demonstrate any risks associated with your application/deployment
-- write a clear and understandable `README` which explains your application and its deployment steps
+- write a clear and understandable `README` which explains your application and its packaging steps, as well as how it meets requirements
```
Comment (felix) [inline README.md]:
```suggestion
- write a clear and understandable `README` which explains your application, its packaging steps, and how it meets the requirements
```

**Review by felix — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (liamn-myob) [reply README.md]: thanks

File: README.md [hunk 4]
```diff
@@ -23,13 +23,13 @@ provider. It should implement the following:
...
 ## Fit and Finish
 
 Once the application has been written, continue with the following additions:
 
 - provide a means of packaging your application as a single deployable artifact which encapsulates its dependencies
-- create a pipeline that builds your application on each commit; Travis or similar, for example
+- create a pipeline that builds your application on each commit; Travis, Github Actions, or similar
 - describe or demonstrate any risks associated with your application/deployment
-- write a clear and understandable `README` which explains your application and its deployment steps
+- write a clear and understandable `README` which explains your application, its packaging steps, and how it meets the requirements
```
Comment (ggershoni) [inline README.md]: and how to run it

**Pull request merged**